### PR TITLE
Composite datastore scrollbar

### LINKED ISSUE #733: Missing scrollbar in CompositeDatastoreDialog
If you have a lot of datastores and try to make a composite dialog, then the vertical size (height) of the dialog might be too big for your screen. And the dialog has no scrollbars, so there is no way to go to the bottom of the list.

### PR DESCRIPTION
**Pull request by tomaszguzialek**
Fixes #733

### CODE REVIEW

File: desktop/ui/src/main/java/org/datacleaner/windows/CompositeDatastoreDialog.java [hunk 1]
```diff
@@ -173,7 +175,11 @@ protected JComponent getDialogContent() {
             checkBoxPanel.add(checkBox);
         }
 
-        WidgetUtils.addToGridBag(checkBoxPanel, formPanel, 0, 1, 2, 1);
+        JScrollPane checkBoxScrollPane = WidgetUtils.scrolleable(checkBoxPanel);
```
Comment (kaspersorensen) [inline desktop/ui/src/main/java/org/datacleaner/windows/CompositeDatastoreDialog.java]: final

File: desktop/ui/src/main/java/org/datacleaner/windows/CompositeDatastoreDialog.java [hunk 2]
```diff
@@ -173,7 +175,11 @@ protected JComponent getDialogContent() {
...
+        checkBoxScrollPane.setPreferredSize(new Dimension(checkBoxPanel.getWidth(), 300));
+        checkBoxScrollPane.setVerticalScrollBarPolicy(JScrollPane.VERTICAL_SCROLLBAR_AS_NEEDED);
+        checkBoxScrollPane.setBackground(_outerPanel.getBackground());
```
Comment (kaspersorensen) [inline desktop/ui/src/main/java/org/datacleaner/windows/CompositeDatastoreDialog.java]: Is this really needed? Normally a scrollpane does not fill the content, it only adds the scroll bars on top.

Comment (tomaszguzialek) [inline desktop/ui/src/main/java/org/datacleaner/windows/CompositeDatastoreDialog.java]: Without setting the background color, the panel became white...

**Comment by kaspersorensen** [comment]
OK. Will merge then :-)

**Pull request merged**